frontend/src/lib/AppNode.ts

```diff
@@ -511,29 +504,13 @@ export class AppRoot {
     // Used to find and update the element node specified by this Delta.
     const { deltaPath } = metadata
 
-    // Update Metrics
-    this.metricsMgr.incrementDeltaCounter(getRootContainerName(deltaPath))
-
```

Comment:
All the removed logic from this function is now in `SegmentMetricsManager.handleDeltaMessage`